packages/react-core/src/next/components/Select/examples/Select.md

```diff
@@ -0,0 +1,31 @@
+---
+id: Select
+section: components
+cssPrefix: pf-c-menu
+propComponents: ['Select', SelectGroup, 'SelectOption', 'SelectList']
+beta: true
+---
+
+import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
+
+## Examples
```

Comment:
I would add an example for a Multi select since it s a very common use case.  An example with a grouped select would be useful as well.